docs/source/design.rst

```diff
@@ -124,7 +124,7 @@ worker processes (unlike when using the preload setting, which loads the code in
 the master process).
 
 .. note:: Under Python 2.x, uou need to install the 'futures' package to use
-this feature.
+  this feature.
```

Comment:
Sphinx uses 3 chars as its default indentation rule. Can you change this to

```rst
.. note:: Under Python 2.x, uou need to install the 'futures' package to use
   this feature.
```

Thanks!